cmd/puppeth/wizard.go

```diff
@@ -231,7 +231,7 @@ func (w *wizard) readDefaultFloat(def float64) float64 {
 // line and returns it. The input will not be echoed.
 func (w *wizard) readPassword() string {
 	fmt.Printf("> ")
-	text, err := terminal.ReadPassword(int(syscall.Stdin))
+	text, err := terminal.ReadPassword(syscall.Stdin)
```

Comment:
`syscall.Stdin` is problematic because it's a different type on Linux and Windows. Lets use `int(os.Stdin.Fd())` here which will work on both platforms and won't cause `unconvert` to whine.

Reply:
Same as other.